stablehlo/tests/interpret/composite.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: stablehlo-translate --interpret -split-input-file %s
+// RUN: stablehlo-translate --interpret %s
```

Comment:
Can we interpret the inlined composite as well? I'm forgetting if the last PR added call op support:

```
// RUN: stablehlo-translate --interpret %s
// RUN: stablehlo-translate --stablehlo-replace-composites-with-calls --interpret %s
```

Reply:
I ended up not adding it to keep the PR more focused. I'm happy to add it, but I'm not sure if that test really adds much. Feels like testing MLIR at that point?

Reply:
Discussed offline. I ended up implementing this.